app/assets/javascripts/controllers/report_data_controller.js

```diff
@@ -404,6 +404,7 @@
   ReportDataController.prototype.setExtraClasses = function(viewType) {
     var mainContent = this.$document.querySelector('#main-content');
     var pagination = this.$document.querySelector('#paging_div .miq-pagination');
+    var formDiv = this.$document.querySelector('form');
```

Comment:
I'm worried about `form` being to generic. (I mean, adding any kind of form anywhere in the page could break the logic.)

If this is specifically about tagging, what about something like `#tagging` here,
and

```diff
diff --git a/app/views/layouts/_tag_edit.html.haml b/app/views/layouts/_tag_edit.html.haml
index 1c6a1472e..a3fc6d36f 100644
--- a/app/views/layouts/_tag_edit.html.haml
+++ b/app/views/layouts/_tag_edit.html.haml
@@ -5,4 +5,5 @@
       = _("Tag Assignment (Tags common to all selected items)")
     - else
       = _("Tag Assignment")
-  = react 'TaggingWrapperConnected', { :tags => @tags, :urls => @button_urls }
+  #tagging
+    = react 'TaggingWrapperConnected', { :tags => @tags, :urls => @button_urls }
```

Reply:
(Alternatively, if the idea is this would break in any form, we could match `#form_div`, add a `miq-form` class, or something like that...)

Reply:
Also watch out for the hack here: https://github.com/ManageIQ/manageiq-ui-classic/search?q=sand&unscoped_q=sand that deals with the backgrounds a hacky way.